cli/src/main/java/bisq/cli/table/builder/AbstractTradeListBuilder.java

```diff
@@ -206,12 +214,19 @@ protected void validate() {
                     ? formatToPercent(t.getOffer().getMarketPriceMargin())
```

Comment:
What about [`formatPercentagePrice`](https://github.com/bisq-network/bisq/blob/14008a670a5193175dc578d04968121cf2693e54/core/src/main/java/bisq/core/util/FormattingUtils.java#L218)?

Reply:
The `cli` module cannot use (depend on) anything in the `core` module.  Everything between `cli` and `daemon` is limited to grpc/protobuf requests & responses, and this forces some duplication.  We'll try to minimize the need to duplicate anything in Bisq2.